### PR TITLE
Add `gh-team-whitelist` command line argument

### PR DESCRIPTION
**Pull request by aknysh**
## what
* Add `gh-team-whitelist` command line argument
* Use bigger atlantis logo on the web pages

## why
* Improved security
* Explicitly specify GitHub teams and operations (`plan`, `apply`) that the members of a team are allowed to perform. Only the users from the whitelisted teams will be able to perform the actions
* https://github.com/runatlantis/atlantis/pull/783
* Closes #26

### CODE REVIEW

File: cmd/server.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	homedir "github.com/mitchellh/go-homedir"
+	"github.com/mitchellh/go-homedir"
```
Comment (Nuru) [inline cmd/server.go:23]: Why did you remove `homedir`? It is referenced later in the file, on line 546 https://github.com/cloudposse/atlantis/blob/60e506dca69dc3c8aba1a41562da4fcf5600c1e7/cmd/server.go#L546

File: Makefile [hunk 2]
```diff
@@ -52,7 +63,9 @@ test-coverage-html:
 	go tool cover -html .cover/cover.out
 
 dist: ## Package up everything in static/ using go-bindata-assetfs so it can be served by a single binary
-	rm -f server/static/bindata_assetfs.go && go-bindata-assetfs -pkg static -prefix server server/static/... && mv bindata_assetfs.go server/static
+	rm -f server/static/bindata_assetfs.go && \
+	go-bindata-assetfs -o server/static/bindata_assetfs.go -pkg static -prefix server server/static/... && \
+	go fmt server/static/bindata_assetfs.go
```
Comment (Nuru) [inline Makefile:68]: I'm not sure, but when I was building this manually I got the impression that there was a reason they did not run `go fmt` on `bindata_assetfs.go`. Possibly because it has huge binary constants representing the compressed assets and it makes `diff`s extra hard to read.

File: cmd/server.go [hunk 3]
```diff
@@ -126,6 +128,14 @@ var stringFlags = map[string]stringFlag{
 		description:  "Hostname of your Github Enterprise installation. If using github.com, no need to set.",
 		defaultValue: DefaultGHHostname,
 	},
+	GHTeamWhitelistFlag: {
+		description: "Comma separated list of key-value pairs representing the GitHub teams and the operations that the members of a particular team are allowed to perform. " +
+			"The format is {team}:{command},{team}:{command}, ex. dev:plan,ops:apply,devops:*. " +
+			"This example means to give the users from the 'dev' GitHub team the permissions to execute the 'plan' command, give the 'ops' team the permissions to execute the 'apply' command, " +
+			"and allow the 'devops' team to perform any operation. If this argument is not provided, the default value (*:*) will be used and the default behavior will be to not check permissions " +
+			"and to allow users from any team to perform any operation.",
```
Comment (Nuru) [inline cmd/server.go]: At least where it is easy to do so, please limit line length to 120 characters so that the code can be read on GitHub without horizontal scrolling. Although even this suggested change requires horizontal scrolling as a comment, at least it does not require horizontal scrolling when included as code.
```
		description: "Comma separated list of key-value pairs representing the GitHub teams and the " +
			"operations that the members of a particular team are allowed to perform. " +
			"The format is {team}:{command},{team}:{command}, ex. dev:plan,ops:apply,devops:*. " +
			"This example means to give the users from the 'dev' GitHub team the permissions to execute " +
			"the 'plan' command, give the 'ops' team the permissions to execute the 'apply' command, " +
			"and allow the 'devops' team to perform any operation. If this argument is not provided, " +
			"the default value (*:*) will be used and the default behavior will be to not check permissions " +
			"and to allow users from any team to perform any operation.",
```

Also, please list the valid values of `command`.

File: server/events/vcs/bitbucketcloud/client.go [hunk 4]
```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/runatlantis/atlantis/server/events/models"
-	validator "gopkg.in/go-playground/validator.v9"
+	"gopkg.in/go-playground/validator.v9"
```
Comment (Nuru) [inline server/events/vcs/bitbucketcloud/client.go:13]: Why did you remove the `validator` alias? It is used several times.

**Review by Nuru — CHANGES_REQUESTED** [review]
I expect the removal of package aliases will break things.  I don't know what motivated the removal, but if it was some kind of compilation error than the compilation environment needs to be fixed/upgraded.

File: cmd/server.go (hunk 1 above)
Comment (aknysh) [reply cmd/server.go:23]: those aliases are redundant and were removed by the IDE
the code compiles with and without them

File: Makefile (hunk 2 above)
Comment (aknysh) [reply Makefile:68]: it does not matter.
Works in both cases.

File: cmd/server.go (hunk 3 above)
Comment (aknysh) [reply cmd/server.go]: made the description line length shorter than 120.
but in the original repo, they have lines with more than 180 chars (so this will not help much with readability)

File: server/events/vcs/bitbucketcloud/client.go (hunk 4 above)
Comment (aknysh) [reply server/events/vcs/bitbucketcloud/client.go:13]: Redundant alias, removed by the IDE

File: cmd/server.go (hunk 3 above)
Comment (osterman) [reply cmd/server.go]: I think we should optimize for maintainability with the upstream. Anything we can/should do to reduce conflicts. This means not changing anything cosmetic.

**Review by osterman — APPROVED** [review]
